ldk/javascript/src/webpack/generate-banner.test.ts

```diff
@@ -44,7 +44,7 @@ describe('Generate Banner', () => {
   it('generates banner given valid LdkSettings', () => {
     const actual = getLoopMetadataContent(generateBanner(ldkSettings));
     const expected = {
-      oliveHelpsContractVersion: '0.1.7',
```

Comment:
Why did we downgrade?

Reply:
That was the contract version I was told to have for the OH `0.26.x` release but the other team captains. But I'm not sure why it's a downgrade from before...